tests/layout/test_placement.py

```diff
@@ -174,3 +175,14 @@ def test_failures(self):
 def rando_placer(S_layout, T_layout):
     T_vertices = list(T_layout)
     return {v: [random.choice(T_vertices)] for v in S_layout}
+
+def test_minimize_overlap(): # To test whether an avoidable OverFlowError in _minimize_overlap is skipped
+    dim_a = 10
+    dim_b = 55
+    grid_edges = list(nx.grid_graph([dim_a, dim_b]).edges())
+    diagonal_edges = [((i, j), (i+1, j+1)) for i in range(dim_b-1) for j in range(dim_a-1)] + [((i, j), (i-1, j+1)) for i in range(1, dim_b) for j in range(dim_a-1)]
+    king = nx.Graph(grid_edges+diagonal_edges)
+    T = dnx.zephyr_graph(15)
+    mml.find_embedding(king, T, scale=1)
+    
+test_minimize_overlap()
```

Comment:
I don't think this line is needed, the test should be picked up by pytest, no?
```suggestion
```

Reply:
Sure, done in `01bec47`